src/styles/InstructorStyles/addCourse.less

```diff
@@ -0,0 +1,154 @@
+// <------ Instructor Add Course ------>
+
+// Content -- Section Wrap
+.iac__section__wrap {
+    font-family:  @text__font-family;
+    background-color: #fbfbfb;
+    padding: 2rem 4rem 2rem 4rem;
+    
+    // IAC Container
+    .iac__container {
+        display: flex;
+        justify-content: space-between;
+    }
+
+    // Title / Top Container
+    .iac__title {
+        font-size: @text__font-size--h1;
+        font-family: @text__font-family--header;
+        font-weight: 300;
+    }
+
+    // Intro
+    .iac__intro {
+        font-size: @text__font-size--h5;
+        line-height: @text__line-height--h5;
+    }
+
+    // <---- Form Divs ---->
```

Comment:
I like that you comment the different sections of the form for easy to read for others.